howtos/oem-guide.rst

```diff
@@ -0,0 +1,135 @@
+
+Our continued existence, i.e. ongoing-development & distribution services,
+depend entirely upon good-will and active community participation in our Open Collective membership Tiers.
+As such circumvention of these service orientated tiers (curated release channels / support / membership / etc),
+in any way, would result in no future communication or service to the offending agent.
+We protect, by way of our licensing choices, and consequent code availability, the end-users freedoms.
+That does not oblige any member of The Rockstor Project's team in any way: regardless of Tier membership.
+
+**Given the above freedoms, all-around, we can only request the following:**
+
+1. OEM hardware distributors/redistributors resellers honour our existing sustainability model.
+I.e. they have an appropriate number, given anticipated development team contacts per-year,
+of 'OEM Pre-installer subscription' memberships.
+Rockstor's continued development, and existence,
+depends entirely on interested parties appreciating both past and ongoing development team efforts and associated costs.
+
+The Rockstor Project development team, in-turn,
+will endeavour to prioritise OEM/reseller technical enquiries/requests as resources allow.
+I.e. a pre-installer's enquiries/concerns are assumed to relate to more than one install,
+where-as our non OEM contribution tiers relate to specific installs; i.e:
+
+- a. Stable Updates subscription €24/year.
+- b. Incident-based Support subscription (`Planned <https://rockstor.com/paid_support.html>`_) €240/incident/year.
+- c. OEM Pre-installer subscription (proposed) €960/year.
+
+.. note::
+
+    Each of the above Tiers is independent,
+    (a.) no implied support,
+    (b.) superset of (a.) including a single support incident.
+    (c.) OEM/pre-installer/for-profit entity membership: no implied end-user support.
+
+2. Any pre-install arrangement must honour our default of neither :ref:`update_channels` channel being enabled.
+All non 'rockstor' package upstream updates are available regardless of this selection.
+See: :ref:`updaterockstorwebui`.
+We stand-by the end-users choice to subscribe to either of our update channels, or neither.
+Support requests, of any sort, related to pre-installed instances,
+where this choice was made by the distributor/re-distributor,
+will be ignored.
+
+3. All pre-installs/re-distributions will involve only our
+`downloadable <https://rockstor.com/dls.html>`_ installer images.
+And then only when said installers include Stable Release Candidate 10 or later status 'rockstor' packages.
+With Stable release status (non RC) strongly preferred: when available.
+
+.. warning::
+
+    Apparent Rockstor installs not conforming to all of (1), (2), and (3) above,
+    will be deemed modified:
+    with all the consequent licensing ramifications regarding redistribution of modified code.
+    We digitally sign all released rpm packages,
```

Comment:
Question, this  signing also goes for RC/test RPMs that are released? More for my own education...

Reply:
@Hooverdan96 Yes, that is correct. When we moved to openSUSE as a base OS it was found that the default setting for zypper was to ignore, or warn-against repositories that were not signed. So we had to start signing the repo metadata. A good thing really. We did previously sign our rpms, and do still. And these signings can serve as indicators of the rpm's origin. See also the following in our rockstor-rpmbuild repo README.md:

- https://github.com/rockstor/rockstor-rpmbuild?tab=readme-ov-file#caveats